src/hooks/useRouteAssetId/useRouteAssetId.ts

```diff
@@ -16,13 +16,21 @@ const getRouteAssetId = (pathname: string) => {
   const assetPaths = [
     '/assets/:chainId/:assetSubId/:nftId',
     '/assets/:chainId/:assetSubId',
+    '/wallet/:chainId/:assetSubId/:nftId',
+    '/wallet/:chainId/:assetSubId',
+    '/wallet/accounts/:chainId/:assetSubId/:nftId',
+    '/wallet/accounts/:chainId/:assetSubId',
+    '/wallet/accounts/:chainId/:assetSubId/:nftId',
+    '/wallet/accounts/:chainId/:assetSubId',
```

Comment:
This is borked and will capture the whole `AccountId` part as `AssetId`
<img width="1646" height="454" alt="Image" src="https://github.com/user-attachments/assets/dd4bba55-07de-48e8-90c8-082a1926af4c" />